.circleci/security-quality.yml

```
@@ -0,0 +1,244 @@
+            pip-audit --format json --output pip-audit-report.json || true
+            echo "Pip audit completed"
+      - run:
+          name: Check for known vulnerabilities
+          command: |
+            if [ -f pip-audit-report.json ]; then
+              echo "Checking for high/critical vulnerabilities..."
+              python3 -c "
+              import json
+              with open('pip-audit-report.json') as f:
+                  data = json.load(f)
+              vulns = [v for v in data.get('vulnerabilities', []) 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)